core/vm/gas_table.go

```diff
@@ -441,6 +472,21 @@ func gasCall(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize
 	if gas, overflow = math.SafeAdd(gas, evm.callGasTemp); overflow {
 		return 0, ErrGasUintOverflow
 	}
+	if evm.Accesses != nil {
+		// TODO omit these for call to self?
```

Comment:
not necessary, but nice to have as this will cause a call to `GetTreeKey`

Reply:
good point: it _needs_ to be called, because a contract can call itself with value 0